swap_meet/clothing.py

```diff
@@ -0,0 +1,24 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=None):
+        self.category = "Clothing"
+        if condition == None:
+            self.condition = 0
+        else:
+            self.condition = float(condition)
```

Comment:
I really like the conditional to confirm and ensure `condition` becomes a float. However, right now you're creating a new instance attribute for Clothing when you can actually inherit it from `Clothing` using the `super` constructor and also move the conditional to check for the `condition` into `Item`.

```suggestion
class Clothing(Item):
    def __init__(self, condition=None):
        super().__init__(self, category = "Category", condition)
```